### PR TITLE
fixed broken links

### LINKED ISSUE #2328: Link to Python models best practices does not exists
[The link to the Python models best practices does not exist.](https://github.com/dbt-labs/docs.getdbt.com/blob/d70b04337bbe59578ddf1fcca1c523f246bf6b6e/website/docs/docs/build/python-models.md?plain=1#L15).

Does anyone know which discussion it should link to?

### PR DESCRIPTION
**Pull request by mirnawong1**
Resolves #2328

### CODE REVIEW

File: website/docs/docs/building-a-dbt-project/building-models/python-models.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dbt Core v1.3 included first-ever support for Python models. Note that only [spe
 
 We encourage you to:
 - Read [the original discussion](https://github.com/dbt-labs/dbt-core/discussions/5261) that proposed this feature.
-- Contribute to [best practices for developing Python models in dbt](https://github.com/dbt-labs/docs.getdbt.com/discussions/1811).
+- Contribute to [best practices for developing Python models in dbt](https://discourse.getdbt.com/c/discussions/21).
```
Comment (jasnonaz) [inline website/docs/docs/building-a-dbt-project/building-models/python-models.md]: We'll want a link to an actual discussion not just the category - @dbeatty10 can help get this set up.

**Review by jasnonaz — CHANGES_REQUESTED** [review]
Thanks for catching this! Just one tweak around making sure we're linking to an actual thread rather than the category itself

**Review by matthewshaver — DISMISSED** [review]

**Review by lostmygithubaccount — DISMISSED** [review]

File: website/docs/docs/building-a-dbt-project/building-models/python-models.md (hunk 1 above)
Comment (lostmygithubaccount) [reply website/docs/docs/building-a-dbt-project/building-models/python-models.md]: created https://discourse.getdbt.com/t/dbt-python-model-dbt-py-best-practices/5204, updating the PR

**Review by lostmygithubaccount — APPROVED** [review]

**Review by dbeatty10 — APPROVED** [review]
Verified that the new link goes to the newly created discussion in Discourse.

Relevant page updated by this PR:
- [docs/building-a-dbt-project/building-models/python-models](https://deploy-preview-2332--docs-getdbt-com.netlify.app/docs/building-a-dbt-project/building-models/python-models)

**Pull request merged**